frontend/webpack.config.js

```diff
@@ -27,6 +27,14 @@ module.exports = (_env, argv) => ({
             test: /\.yaml$/u,
             use: "yaml-loader",
             type: "json",
+        }, {
+            test: /\.svg$/u,
+            use: [{
+                loader: "@svgr/webpack",
+                options: {
+                    icon: true,
+                },
+            }],
```

Comment:
This works, of course, and if we specify the correct ambient type definition (see above), we even have it type-safe. However, doesn't it suck that this isn't handled automatically? svgr can actually generate TypeScript components. The Webpack loader just wraps this appropriately, which is nice, but has the disadvantage that `tsc` never gets to see them. However, we could actually generate them manually and `import` the resulting files in our code, similar to what we do with `relay-compiler`.

This doesn't have to go in this PR, but if there is no opposition, I would like to try to get this working. Opinions?

Reply:
Well... I really don't see it as a serious issue. "Slightly meh" probably captures my feelings best. (Again, under the assumption that all SVG files have the same type and that we get the same type safety with the ambient type declaration above.) Thus, I really wouldn't put more than "a little" time into improving this situation.